packages/aws-cdk/lib/api/deployments/asset-publishing.ts

```diff
@@ -241,10 +183,10 @@ export const EVENT_TO_LOGGER: Record<EventType, (x: string) => void> = {
 };
 
 class PublishingProgressListener implements IPublishProgressListener {
-  constructor(private readonly quiet: boolean) {}
+  constructor() {}
 
   public onPublishEvent(type: EventType, event: IPublishProgress): void {
-    const handler = this.quiet && type !== 'fail' ? debug : EVENT_TO_LOGGER[type];
+    const handler = EVENT_TO_LOGGER[type];
```

Comment:
Removed unused `quiet` option upstream in favor of it always being the default (`false`) => we just use the default part of the conditional.